projects/find/find.py

```diff
@@ -0,0 +1,57 @@
+import os
+import sys
+
+"""this method is created to find all file and directory, given a specific path. Optionally, your can use -name flag
+to search only the files that match to a specific name"""
+
+
+def find(path, option=None, filename=None):
+    """this method uses a bfs approach to resolve to the problem of finding all files and directories."""
+    if not os.path.exists(path):
+        return "Path non corretto o inestitente: si prega di riprovare"
+    queue = []
+    queue.append(path)
+    # marked = [False] * len(list)
+    # edge_to = [None] * len(list)
+    list = []
```

Comment:
`list` is a reserved keyword in python. you can assign to it, but you will override it's default behaviour. also, `list` doesn't mean much in this context, as i can't understand what this list contains. so please find a better name for it